### PR TITLE
fleet list

### PR DESCRIPTION
**Pull request by danh132**
fleet list - show all fleet in airlines, and map picture with large pdf
fleet

### CODE REVIEW

File: app/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -170,6 +170,9 @@ private function mapWebRoutes()
                 Route::get('pilots', 'UserController@index')->name('pilots.index');
 
                 Route::get('livemap', 'LiveMapController@index')->name('livemap.index');
+                
+                Route::get('showFleet', 'FleetController@showFleet')->name('fleet.show');
```
Comment (nabeelio) [inline app/Providers/RouteServiceProvider.php:174]: `show_fleet`

File: resources/views/layouts/default/flights/fleet.blade.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
+@extends('app')
+@section('title', __('home.welcome.title'))
```
Comment (nabeelio) [inline resources/views/layouts/default/flights/fleet.blade.php:2]: Change the title. This file also needs to be formatted properly like the other templates are formatted - 2 space indents, etc

File: resources/views/layouts/default/flights/fleet.blade.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  @endforeach
+  </tbody>
+</table>
+
+        
+</div>
+     
+     <div class="col-sm-6">
+     <h2 class="description">Our destinations</h2>
+     <a href="Routes.pdf" title="Click for zoom"><img src="Routes.png"></a>
```
Comment (nabeelio) [inline resources/views/layouts/default/flights/fleet.blade.php:59]: What is this image and PDF file?

**Comment by nabeelio** [comment]
StyleCI fixes need to be applied, click "Details" to see what needs to be fixed/formatted

File: resources/views/layouts/default/flights/fleet.blade.php (hunk 3 above)
Comment (danh132) [reply resources/views/layouts/default/flights/fleet.blade.php]: image of existing routes - small image and then large resolution pdf

File: resources/views/layouts/default/flights/fleet.blade.php (hunk 2 above)
Comment (danh132) [reply resources/views/layouts/default/flights/fleet.blade.php]: done

File: app/Providers/RouteServiceProvider.php (hunk 1 above)
Comment (danh132) [reply app/Providers/RouteServiceProvider.php]: underscore also in controller ?

**Comment by danh132** [comment]
what is problem with StyleCI now ?

**Comment by FatihKoz** [comment]
style_errors

And would you mind explaining where and how the route.png file (you are referring in the blade) is created ? I saw no code generating it in the pr/changed files.

Also as a thought, i think in $aircraft should be $aircrafts 'cause it holds multiple aircraft entries ;) $aircraft is kind of misleading.

**Pull request closed without merging**